test/Microsoft.ML.Tests/Microsoft.ML.Tests.csproj

```diff
@@ -23,9 +24,6 @@
     <NativeAssemblyReference Include="FastTreeNative" />
     <NativeAssemblyReference Include="LdaNative" />
 	<NativeAssemblyReference Include="SymSgdNative" />
-  </ItemGroup>
-  
-  <ItemGroup>
-    <PackageReference Include="MlNetMklDeps" Version="$(MlNetMklDepsPackageVersion)" />
+    <NativeAssemblyReference Include="MklImports" />
```

Comment:
> <NativeAssemblyReference Include="MklImports" /> [](start = 3, length = 49)

curious, isn't the reference ot the HalProject enough to drag the mkl dependency?

Reply:
No.  The only way SymSGD can work is when libMklImports assembly is right next to SymSGD.  So this `NativeAssemblyReference` will copy libMklImports into the test output folder right next to SymSGD.